core/app/modules/import-export/assets/js/shared/kit-content/kit-content.js

```diff
@@ -50,7 +54,24 @@ export default function KitContent( { contentData, hasPro } ) {
 									onMouseLeave={ () => isLockedFeaturesNoPro && setContainerHoverState( index, false ) }
 								>
 									<Grid container noWrap >
-										<KitContentCheckbox type={ type } className="e-app-export-kit-content__checkbox" />
+										<KitContentCheckbox
+											type={ type }
+											className="e-app-export-kit-content__checkbox"
+											referrer={ referrer }
```

Comment:
We pass the referrer a lot.
It was better to include it in a context.

Let's discuss it as part of step 2.

Reply:
o.k